docs/pages/setup/security/reduce-blast-radius.mdx

```diff
@@ -0,0 +1,187 @@
+
+
+## Require multiple reviews for role requests
+Even if an attacker gains access to a user's credentials and successfully signs into your Teleport cluster, you can still prevent the user from escalating their privileges. If you enable dual authorization, users who request to assume a particular role must obtain permission to do so from two or more reviewers. This way, if a malicous user manages to impersonate a legitimate one, reviewers can contact the real user before granting the new role.
+
+Dual authorization uses Teleport's access plugins—e.g., Slack, JIRA, and PagerDuty—to notify reviewers that a user has requested a role. For access plugins that require a SAML or OIDC connector, you must enable the Cloud or Enterprise versions of Teleport.
+
+You can set up dual authorization by applying two dynamic resources:
+
+- [The reviewer](#the-reviewer)
+- [The reviewee](#the-reviewee)
+
+### The reviewer
+You can enable some users to review other users' role escalation requests by applying a dynamic resource similar to the following:
+
+```yaml
+kind: role
+version: v4
+metadata:
+  name: reviewer
+spec:
+  allow:
+    review_requests:
+      roles: ["role-one", "role-two", "role-three"]
+```
+
+Assign `spec.allow.review_requests.roles` to a list of role names. When a user requests access to one of the roles listed in `spec.allow.review_requests.roles`, your Teleport access plugins notify users with the `reviewer` role of the request and relay the responses to your Teleport cluster.
+
+### The reviewee
+You can require a user to request access from reviewers by applying a dynamic resource similar to the following:
+
+```yaml
+kind: role
+version: v4
+metadata:
+  name: reviewee
+spec:
+  allow:
+    request:
+      roles: ["role-one", "role-two", "role-three"]
+      thresholds:
+        - approve: 2
+          deny: 1
+```
+
+The `spec.allow.request.roles` field lists the names of other roles that a user with the `reviewee` role can request. When a reviewee requests access to one of these roles, Teleport notifies reviewers via your access plugins. The `spec.allow.requests.roles.thresholds` field indicates how many reviews are required to approve or deny the request.
+
+
+## Automatically prevent some roles from requesting others
+In theory, a malicious Teleport user could request a more privileged role and trick a reviewer into granting access. You can prevent such a scenario by defining roles that prohibit users from even requesting access to particular roles.
+
+The `spec.deny` field has the same possible properties as the `spec.allow` field we described [earlier](#require-multiple-reviews-for-role-requests) except, rather than enabling actions, this field disables them. For example, the `spec.deny.requests.roles` field is a list of roles that a user is prohibited from requesting.
```

Comment:
Might be helpful to mention here that "deny" rules take precedence if a user has multiple roles assigned.

Reply:
👍